src/codemagic/tools/universal_apk_generator.py

```diff
@@ -105,15 +105,18 @@ def generate(self) -> List[pathlib.Path]:
         from .android_app_bundle import AndroidAppBundleTypes
 
         self._deprecation_notice()
-        signing_info_kwargs = {}
-        if self.android_signing_info:
-            signing_info_kwargs = {
-                'keystore_path': self.android_signing_info.store_path,
-                'keystore_password': AndroidAppBundleTypes.KeystorePassword(self.android_signing_info.store_pass),
-                'key_alias': AndroidAppBundleTypes.KeyAlias(self.android_signing_info.key_alias),
-                'key_password': AndroidAppBundleTypes.KeyPassword(self.android_signing_info.key_pass),
-            }
-        return AndroidAppBundle().build_universal_apks(self.pattern, **signing_info_kwargs)
+        android_app_bundle = AndroidAppBundle()
+        return (
+            android_app_bundle.build_universal_apks(
+                self.pattern,
+                keystore_path=self.android_signing_info.store_path,
+                keystore_password=AndroidAppBundleTypes.KeystorePassword(self.android_signing_info.store_pass),
+                key_alias=AndroidAppBundleTypes.KeyAlias(self.android_signing_info.key_alias),
+                key_password=AndroidAppBundleTypes.KeyPassword(self.android_signing_info.key_pass),
+            )
+            if self.android_signing_info
+            else android_app_bundle.build_universal_apks(self.pattern)
+        )
```

Comment:
```suggestion
        if self.android_signing_info:
            apk_paths = android_app_bundle.build_universal_apks(
                self.pattern,
                keystore_path=self.android_signing_info.store_path,
                keystore_password=AndroidAppBundleTypes.KeystorePassword(self.android_signing_info.store_pass),
                key_alias=AndroidAppBundleTypes.KeyAlias(self.android_signing_info.key_alias),
                key_password=AndroidAppBundleTypes.KeyPassword(self.android_signing_info.key_pass),
            )
        else:
            apk_paths = android_app_bundle.build_universal_apks(self.pattern)
        return apk_paths
```

I find that it is easier to follow what is going on if the conditional statement is preceeding the statement.